.github/workflows/image_base.yml

```diff
@@ -1,9 +1,7 @@
 name: base
 
 on:
-  push:
-    branches: 
-      - 'base'
+  workflow_dispatch:
```

Comment:
base image build needs to be triggered manually now?

Reply:
the multi arch image build is still not working. Let's create a separate PR for that. For now, I just made this a manual workflow.

Reply:
> the multi arch image build is still not working

can you point me to the pipeline?

Reply:
https://github.com/sustainable-computing-io/kepler/blob/main/.github/workflows/image_base.yml